### PR TITLE
[Refactor: design sdk_client_interface & rename zk_client.]

### PR DESCRIPTION
**Pull request by lishuo5263**
* Design sdk_client_interface
* rename zk_client

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 English | [简体中文](README_CN.md)
 
+[![OSCS Status](https://www.oscs1024.com/platform/badge/apache/incubator-shenyu-client-golang.svg?size=small)](https://www.oscs1024.com/project/apache/incubator-shenyu-client-golang?ref=badge_small)
```
Comment (yu199195) [inline README.md:5]: remove this

File: README_CN.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 English | [简体中文](README_CN.md)
 
+[![OSCS Status](https://www.oscs1024.com/platform/badge/apache/incubator-shenyu-client-golang.svg?size=small)](https://www.oscs1024.com/project/apache/incubator-shenyu-client-golang?ref=badge_small)
```
Comment (yu199195) [inline README_CN.md:5]: remove this

File: README.md (hunk 1 above)
Comment (lishuo5263) [reply README.md]: it been removed

File: README_CN.md (hunk 2 above)
Comment (lishuo5263) [reply README_CN.md]: it been removed

**Review by yu199195 — APPROVED** [review]

**Pull request merged**